### ISSUE TITLE
What do you mine on the processor, please advise?

### ISSUE THREAD

**Issue by vavilovalila** [UnamSanctam/SilentCryptoMiner#1840]
What do you mine on the processor, please advise?

**Comment by UnamSanctam** [comment]
Monero is the common CPU mining coin.

**Comment by Aciid-Nuk3r** [comment]
> What do you mine on the processor, please advise?

Are you referring to what is profitable on the CPU or what the capabilities for mining are?

**Comment by vavilovalila** [comment]
> > What do you mine on the processor, please advise?
> 
> Are you referring to what is profitable on the CPU or what the capabilities for mining are?

more profitable

**Comment by Aciid-Nuk3r** [comment]
> > > What do you mine on the processor, please advise?
> 
> > 
> 
> > Are you referring to what is profitable on the CPU or what the capabilities for mining are?
> 
> 
> 
> more profitable

Then that would depend entirely upon the CPU power you have as it stands, Monero (XMR) is quite profitable.

Again, I'll reiterate, the profitability is dependent upon the overall processing power (Hash rate) you can generate.

Actually, the Monero team has made it difficult for ASICS based miners to mine XMR, they want it to be more CPU focused.

- XMR
- QuazarCoin
- ByteCoin

TL;DR

I would recommend XMR because Monero has a constantly changing algorithm and it stands in the CPU's favor for mining and profitability.
